stratum/hal/lib/tdi/tdi_sde_wrapper.h

```diff
@@ -25,6 +23,10 @@
 #include "stratum/hal/lib/common/common.pb.h"
 #include "stratum/lib/channel/channel.h"
 
+#ifdef TOFINO_TARGET
```

Comment:
Can we move this into tdi_sde_wrapper.cc? I am not sure why it is even here in the first place.

Reply:
No, we can't. The #ifdef is here because the %$@#! Packet I/O methods were defined using Tofino-specific symbols. They should be replaced with abstract interfaces that are SDE-neutral.

This is why I asked about the Packet I/O interface earlier today. I really want the TDI team to fix this.